relayer/chains/evm/events.go

```diff
@@ -10,11 +10,13 @@ import (
 
 // All the events
 var (
-	EmitMessage = "Message(string,uint256,bytes)"
-	CallMessage = "CallMessage(string,string,uint256,uint256,bytes)"
+	EmitMessage     = "Message(string,uint256,bytes)"
+	CallMessage     = "CallMessage(string,string,uint256,uint256,bytes)"
+	ExecuteRollback = "ExecuteRollback(uint256)"
```

Comment:
perhaps this should also be RollbackMessage similar to other messages.

Reply:
yep, nice catch.